ramls/settings/authoritysourcefile.json

```diff
@@ -13,6 +13,7 @@
     },
     "codes": {
       "type" : "array",
+      "description": "array of codes",
```

Comment:
@RomanChernetskyi should we add these properties manually? Can we just update raml-storage?

Reply:
Discussed with Roman - everything ok. We need to update it manually. Resolve conversation.

Reply:
Hi, we should add these properties manually, because these files are not from raml-storage